tfhe/src/integer/server_key/radix_parallel/div_mod.rs

```diff
@@ -125,25 +115,15 @@ impl ServerKey {
                     &merge_two_cmp_lut,
                 );
 
-                interesting_divisor.blocks.par_iter_mut().for_each(|block| {
-                    self.key.unchecked_apply_lookup_table_bivariate_assign(
-                        block,
-                        &merged_cmp,
-                        &zeroer_lut_for_merged_cmp,
-                    );
-                });
+                self.zero_out_if_condition_is_false(&mut interesting_divisor, &merged_cmp);
```

Comment:
more efficient ?

Reply:
No its the same thing but hopefully more readable